README.md

```diff
@@ -1,11 +1,93 @@
-The plugin is inspired from the Matrix Reloaded Plugin and allows the user to choose which matrix combinations he
-wants to run.
+This plugin allows a user to choose which matrix combinations he wants to run, as opposed to the default behaviour where jenkins runs all combinations and translate it into a groovy condition such as:
 
-This plugin add a new type of parameter.
-When the user build a matrix project with this parameter type he gets a html table with checkbox for each combination.
-It gets from the user which combinations he wants to run, and translate it into a groovy condition such as:
+axis1=="axis1value1" && axis2=="axis2value1" || axis1=="axis1value2" && axis2=="axis2value2" || ...:
 
-axis1=="axis1value1" && axis2=="axis2value1" || axis1=="axis1value2" && axis2=="axis2value2" || ...
+  
+Using:
 
+After clicking the build, the user gets a html table with a checkbox for each matrix combination.
 
+After clicking the submit button, only checked combinations will run.
 
+![](https://wiki.jenkins.io/download/attachments/69271855/MatrixJobConfig.png?version=1&modificationDate=1375012523000&api=v2)
+
+Configuration:
+
+After installation a new type of parameter was added (Matrix Combinations Parameter).
+
+ 
+![](https://wiki.jenkins.io/download/attachments/69271855/MatrixPluginConfgJob.png?version=1&modificationDate=1375012524000&api=v2)
+
+-   Configure your matrix job and add exactly one parameter with this type.
+-   Fill the name of the parameter.
+-   Define the Axes as usual
+
+# **Acknowledgements**
```

Comment:
# is a top level heading (title) and should only be on the first line of the file